frontend/src/locales/en.json

```diff
@@ -1,355 +1,358 @@
-    "Let the price move with the market":"Let the price move with the market",
-    "Premium over Market (%)":"Premium over Market (%)",
-    "Explicit":"Explicit",
-    "Set a fix amount of satoshis":"Set a fix amount of satoshis",
-    "Satoshis":"Satoshis",
-    "Let the taker chose an amount within the range":"Let the taker choose an amount within the range",
-    "Enable Amount Range":"Enable Amount Range",
+    "Order": "Order",
+    "Customize": "Customize",
+    "Buy or sell bitcoin?": "Buy or sell bitcoin?",
+    "Buy": "Buy",
+    "Sell": "Sell",
+    "Amount": "Amount",
+    "Amount of fiat to exchange for bitcoin": "Amount of fiat to exchange for bitcoin",
+    "Invalid": "Invalid",
+    "Enter your preferred fiat payment methods. Fast methods are highly recommended.": "Enter your preferred fiat payment methods. Fast methods are highly recommended.",
+    "Must be shorter than 65 characters": "Must be shorter than 65 characters",
+    "Swap Destination(s)": "Swap Destination(s)",
+    "Fiat Payment Method(s)": "Fiat Payment Method(s)",
+    "You can add any method": "You can add any method",
+    "Add New": "Add New",
+    "Choose a Pricing Method": "Choose a Pricing Method",
+    "Relative": "Relative",
+    "Let the price move with the market": "Let the price move with the market",
+    "Premium over Market (%)": "Premium over Market (%)",
+    "Explicit": "Explicit",
+    "Set a fix amount of Sats": "Set a fix amount of Sats",
+    "Sats": "Sats",
+    "Let the taker chose an amount within the range": "Let the taker choose an amount within the range",
+    "Enable Amount Range": "Enable Amount Range",
     "From": "From",
-    "to":"to",
-    "Public Duration (HH:mm)":"Public Duration (HH:mm)",
-    "Set the skin-in-the-game, increase for higher safety assurance":"Set the skin-in-the-game, increase for higher safety assurance",
-    "Fidelity Bond Size":"Fidelity Bond Size",
-    "Allow bondless takers":"Allow bondless takers",
-    "COMING SOON - High risk! Limited to {{limitSats}}K Sats":"COMING SOON - High risk! Limited to {{limitSats}}K Sats",
-    "You must fill the order correctly":"You must fill the order correctly",
-    "Create Order":"Create Order",
-    "Back":"Back",
-    "Create a BTC buy order for ":"Create a BTC buy order for ",
-    "Create a BTC sell order for ":"Create a BTC sell order for ",
-    " of {{satoshis}} Satoshis":" of {{satoshis}} Satoshis",
-    " at market price":" at market price",
-    " at a {{premium}}% premium":" at a {{premium}}% premium",
-    " at a {{discount}}% discount":" at a {{discount}}% discount",
-    "Must be less than {{max}}%":"Must be less than {{max}}%",
-    "Must be more than {{min}}%":"Must be more than {{min}}%",
+    "to": "to",
+    "Public Duration (HH:mm)": "Public Duration (HH:mm)",
+    "Set the skin-in-the-game, increase for higher safety assurance": "Set the skin-in-the-game, increase for higher safety assurance",
+    "Bond size": "Bond size",
```

Comment:
I agree harmonization is good. Yet, context is important. This is the first time many users will see the word "bond". And it is good there is an extra pinch of information of what sort of bond it is: it is a "fidelity bond" (user reaction: wait, what okay? Fidelity? ...I guess that has to do with not cheating? I get it!)

Reply:
Instead adding an adjective to the naming, why not use this way to show under the %, some additional info like in this step?

![image](https://user-images.githubusercontent.com/68381662/163722186-78c74ba6-e88b-48f0-9cdc-f45ac5a26cfc.png)

![image](https://user-images.githubusercontent.com/68381662/163722167-f26b539c-0c88-4fec-8438-8b38497e3e32.png)

This text can fit (and you can remove the tooltip):

`The bond is a safety assurance that sets the peers skin-in-the game.`

Reply:
It's a great idea. We can add a [(?)](https://mui.com/material-ui/material-icons/?query=info&selected=HelpCenter) icon and on click/tap/hover it can display a tooltip just like the one on the second screenshot!